src/components/Modal/README.md

```diff
@@ -1065,27 +1065,29 @@ accessibility.
-| `--rui-Modal--fullscreen__width`                     | Width of fullscreen modal                                     |
-| `--rui-Modal--fullscreen__height`                    | Height of fullscreen modal                                    |
+| Custom Property                                      | Description                                                 |
+|------------------------------------------------------|-------------------------------------------------------------|
+| `--rui-Modal__padding-x`                             | Inline padding of individual modal components               |
+| `--rui-Modal__padding-y`                             | Block padding of individual modal components                |
+| `--rui-Modal__background`                            | Modal background (including `url()` or gradient)            |
+| `--rui-Modal__box-shadow`                            | Modal box shadow                                            |
+| `--rui-Modal__separator__width`                      | Width of separator between modal header, body, and footer   |
+| `--rui-Modal__separator__color`                      | Color of separator between modal header, body, and footer   |
+| `--rui-Modal__outer-spacing-xs`                      | Spacing around modal, `xs` screen size                      |
+| `--rui-Modal__outer-spacing-sm`                      | Spacing around modal, `sm` screen size and bigger           |
+| `--rui-Modal__header__gap`                           | Modal header gap between children                           |
+| `--rui-Modal__footer__background`                    | Modal footer background (including `url()` or gradient)     |
+| `--rui-Modal__footer__gap`                           | Modal footer gap between children                           |
+| `--rui-Modal__backdrop__background`                  | Modal backdrop background (including `url()` or gradient)   |
+| `--rui-Modal--auto__min-width`                       | Min width of auto-sized modal (when enough screen estate)   |
+| `--rui-Modal--auto__max-width`                       | Max width of auto-sized modal (when enough screen estate)   |
+| `--rui-Modal--small__width`                          | Width of small modal                                        |
+| `--rui-Modal--medium__width`                         | Width of medium modal                                       |
+| `--rui-Modal--large__width`                          | Width of large modal                                        |
+| `--rui-Modal--fullscreen__width`                     | Width of fullscreen modal                                   |
+| `--rui-Modal--fullscreen__height`                    | Height of fullscreen modal                                  |
+| `--rui-Modal__animation__duration`                   | Duration of animation used (when opening modal)             |
```

Comment:
> when opening modal

Currently, closing of `Modal` is not animated because the component is simply removed from the DOM.

@mbohal Do we also want animated closing? That would mean delayed removal from the DOM.

Reply:
I would like to add one thing. Closing animation is hard to impelement. We do not have close handler, we have `closeButtonRef`. I tried to implement it somehow today but without success. So it is up to our decision whether we want to spend some time on it or not.

Reply:
After a fruitless afternoon I concur with @bedrich-schindler that:

> Closing animation is hard to implement.

However, this change is not BC in terms of usage, so I think we should merge it as is.

We can add hide animation int he future. This will very likely change the API though…

Reply:
On video call agreed that for the time being we will only support opening animation.